### DISCUSSION TITLE
Add Intel's IPEX-LLM

### DISCUSSION THREAD

**Discussion by NikosDi** [janhq/jan#4019] [Feature Ideas]
Hi.
I think it would be useful to add more GPU accelerated engines beyond nVidia CUDA.

Intel's IPEX-LLM (Intel Extension for PyTorch) is a good example.

**Comment by poroburu** [comment]
https://jan.ai/docs/local-engines/llama-cpp
https://ipex-llm-latest.readthedocs.io/en/latest/doc/LLM/Quickstart/llama_cpp_quickstart.html

It looks like the local engine functionality got pushed into cortex llama.cpp wrapper in development?
https://cortex.so/docs/engines/engine-extension
